cpp/src/parquet/bloom_filter_reader_writer_test.cc

```diff
@@ -0,0 +1,216 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
This is moved from `bloom_filter_reader_test.cc`

Reply:
I don't know if github can be smart about it but was git mv used?  I thought that might limit the diff?